Core.DomainModel/ItSystemUsage/ItSystemUsage.cs

```diff
@@ -891,10 +899,16 @@ public Maybe<OperationError> UpdateDataSensitivityLevels(IEnumerable<SensitiveDa
                 ItSystemUsage = this,
                 SensitivityDataLevel = sensitiveDataLevel
             }).ToList();
+            
+            var removedPersonalData = new List<ItSystemUsagePersonalData>();
+            if (levels.Contains(SensitiveDataLevel.PERSONALDATA) == false)
```

Comment:
once you have mirrored you can actually use the helper in stead of this... and then clear the persondata

Reply:
```
        public Result<IEnumerable<ItSystemUsagePersonalData>, OperationError> UpdateDataSensitivityLevels(IEnumerable<SensitiveDataLevel> sensitiveDataLevels)
        {
            if (sensitiveDataLevels == null)
                throw new ArgumentNullException(nameof(sensitiveDataLevels));

            var levels = sensitiveDataLevels.ToList();

            if (levels.Distinct().Count() != levels.Count)
                return new OperationError("Duplicate sensitivity levels are not allowed", OperationFailure.BadInput);

            var levelMappings = levels.Select(sensitiveDataLevel => new ItSystemUsageSensitiveDataLevel()
            {
                ItSystemUsage = this,
                SensitivityDataLevel = sensitiveDataLevel
            }).ToList();

            var removedPersonalData = new List<ItSystemUsagePersonalData>();

            levelMappings.MirrorTo(SensitiveDataLevels, x => x.SensitivityDataLevel);

            if (!SensitiveDataLevelExists(SensitiveDataLevel.PERSONALDATA))
            {
                removedPersonalData.AddRange(ResetPersonalData());
            }

            return removedPersonalData;
        }

```